server/parsec/cli/run.py

```diff
@@ -292,20 +279,15 @@ def handle_parse_result(
     help="Sender address used in sent emails",
 )
 @click.option(
-    "--forward-proto-enforce-https",
-    type=str,
-    show_default=True,
-    default=None,
-    callback=lambda ctx, param, value: _parse_forward_proto_enforce_https_check_param(value),
-    envvar="PARSEC_FORWARD_PROTO_ENFORCE_HTTPS",
+    "--proxy-trusted-addresses",
+    default=[],
+    envvar="PARSEC_PROXY_TRUSTED_ADDRESSES",
+    callback=lambda ctx, param, value: str(value).split(","),
     show_envvar=True,
-    help=(
-        "Enforce HTTPS by redirecting incoming request that do not comply with the provided header."
-        " This is useful when running Parsec behind a forward proxy handing the SSL layer."
-        " You should *only* use this setting if you control your proxy or have some other"
-        " guarantee that it sets/strips this header appropriately."
-        " Typical value for this setting should be `X-Forwarded-Proto:https`."
-    ),
+    help="""\b
+        Comma-separated list of IP Addresses, IP Networks or literals to trust with proxy headers.
+        Settings this value will allow the server to use the forwarded headers from those clients.
```

Comment:
```suggestion
        Set this value to allow the server to use X-Forwarded-* headers from those clients.
```

Reply:
```suggestion
        Set this value to allow the server to use the forwarded headers from those clients.
```

Don't specify `X-Forwarded` in case `uvicorn` add support for it.